Add Fleet & Agent 8.13.3 Release Notes (#1052)
elastic/ingest-docs: docs/en/ingest-management/release-notes/release-notes-8.13.asciidoc

* {kibana-ref}/release-notes.html[{kib} release notes]
* {beats-ref}/release-notes.html[{beats} release notes]

// begin 8.13.3 relnotes

[[release-notes-8.13.3]]
== {fleet} and {agent} 8.13.3

Review important information about {fleet-server} and {agent} for the 8.13.3 release.

[discrete]
[[security-updates-8.13.3]]
=== Security updates

{agent}::
* Update Go version to 1.21.9. {agent-pull}4508[#4508]

[discrete]
[[bug-fixes-8.13.3]]
=== Bug fixes

{fleet}::
* Fix managed agent policy preconfiguration update. ({kibana-pull}181624[#181624])
* Use lowercase dataset name in template and pipeline names. ({kibana-pull}180887[#180887])
* Fix KQL/kuery for getting {fleet-server} agent count. ({kibana-pull}180650[#180650])

{agent}::

> * Reduce false positives in logging an API switch request from {fleet-server}. {agent-pull}4481[#4481] {agent-issue}4477[#4477]

Reviewer: ```suggestion
* Fix unconditionally logging all 400 HTTP responses from {fleet-server} as API compatibility errors. {agent-pull}4481[#4481] {agent-issue}4477[#4477]
```

> * Fix failing upgrade command when the connection to an {agent} is disconnected. {agent-pull}4519[#4519] {agent-issue}3890[#3890s]

Reviewer: ```suggestion
* Allow the `elastic-agent upgrade` command to succeed when the agent restarts before it has completely acknowledged the upgrade. {agent-pull}4519[#4519] {agent-issue}3890[#3890s]
```

> 

Reviewer: ```suggestion

* Only allow installing Elastic Defend on Windows kernels newer than 6, i.e. Windows 10 /
Server 2016 and newer
```

Looks like https://github.com/elastic/elastic-agent/commit/0065ee336775b99d6407cc80eae1d163587e30b6 is missing, I know this has caused some problems for security users so we should include it as well.
